syntax = "proto3";

package order_executor;

service OrderExecutorService {
}